### PR TITLE
Completed CV Bootcamp

### PR DESCRIPTION
**Pull request by tmaswang03**
Includes plot.png plot of validation loss + training loss vs epoch as well as terminal output of the program in terminal_output.txt and the program itself in main.py.

### CODE REVIEW

**Comment by Trotyl15** [comment]
Your code looks good overall. Welcome onboard!
